modules/ocf_mirrors/manifests/kde_appdata.pp

```diff
@@ -0,0 +1,27 @@
+class ocf_mirrors::kde_appdata {
+  file {
+    '/opt/mirrors/project/kde-appdata':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/kde-appdata/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  ocf_mirrors::monitoring {
+    'kde-appdata':
```

Comment:
I think this needs to be `kde-applicationdata` (or the `local_path` attribute needs to be changed) since the path is `http://mirrors.ocf.berkeley.edu/kde-applicationdata/last-updated`, not `http://mirrors.ocf.berkeley.edu/kde-appdata/last-updated`.